### PR TITLE
ts: props type generic for Controller.getSignal

### PR DESCRIPTION
**Pull request by manneredboor**
Added the generic for make definable type of signal, returning from ControllerClass.getSignal

### CODE REVIEW

File: packages/node_modules/cerebral/index.d.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export interface ControllerClass extends FunctionTree {
   getModel(): StateModel
   getState(path?: string): any
   runSignal(name: string, signal: SignalChain, payload: any): void
-  getSignal(path: string): Signal
+  getSignal<T=any>(path: string): Signal<T>
```
Comment (Guria) [inline packages/node_modules/cerebral/index.d.ts]: What I don'like here about typing systems like ts and flow is that is could be treated as breaking change :(
But bump major version on each such change is overkill

**Comment by gaspard** [comment]
Nice contribution. @manneredboor Can you rephrase your commit so that it matches the pattern (used when publishing):

```
fix(types): some message here
```

**Comment by manneredboor** [comment]
@gaspard Done!
